packages/kbn-babel-preset/common_preset.js

```diff
@@ -32,6 +29,12 @@ module.exports = {
     //
     // See https://github.com/babel/proposals/issues/12 for progress
     require.resolve('@babel/plugin-proposal-class-properties'),
+    [
+      require.resolve('@kbn/babel-plugin-apm-idx'),
+      {
+        importName: /\/common\/idx$/,
```

Comment:
importName now supports a RegExp pattern to test Import Specifiers in order to allow our use case, i.e. relative path imports of `*/common/idx.ts`